1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During DC/OS installation, the components come online in a particular sequence due to interdependencies.
+
+## Master nodes
+
+On each master node the following happens, in chronological order:
+
+1. [Exhibitor](https://github.com/mesosphere/exhibitor-dcos) starts up, creates ZooKeeper configuration and launches ZooKeeper.
+1. The Mesos masters are launched, register with the local ZooKeeper (127.0.0.1), and discover the other masters.
+1. Mesos-DNS is launched on every master node.
+1. Mesos-DNS keeps hitting `leader.mesos:5050/master/state.json`. The DNS entry `leader.mesos` points to the currently leading Mesos master.
+1. The Distributed DNS Proxy runs on all master and agent nodes, and forwards DNS lookups to Mesos-DNS.
+1. DC/OS Marathon is launched and starts on every master node.
+1. DC/OS Marathon connects to the local ZooKeeper (127.0.0.1), discovers the leading Mesos master (`leader.mesos`) and registers as a framework.
+1. Admin Router depends on the Mesos master, Mesos-DNS, and the Distributed DNS Proxy. It runs on each of the master nodes. The admin router is what serves the DC/OS UI and proxies external admin connections into the cluster.
```

Comment:
The rest of the numbered items describe boot sequence actions. The only action described here is admin router depending on the master. Is it started or initiated or something?